website/content/docs/connect/gateways/api-gateway/index.mdx

```diff
@@ -1,28 +1,61 @@
-description: API gateways are objects in Consul that enable ingress requests to services in your service mesh. Learn about API gateways for VMs in this overview.
+page_title: API gateways overview
+description: API gateways provide an ingress point for service mesh traffic. Learn how API gateways add listeners for external traffic and route HTTP requests to services in the mesh.
 ---
 
-# API gateway overview
+# API gateways overview
 
-API gateways enable external network clients to access applications and services
-running in a Consul datacenter. This type of network traffic is commonly
-called _north-south_ network traffic because it refers to the flow of
-data into and out of a specific environment. API gateways can also forward
-requests from clients to specific destinations based on path or request
-protocol.
+This topic provides overview information about API gateways in Consul. API gateways enable external network clients to access applications and services running in a Consul datacenter. Consul API gateways can also forward requests from clients to specific destinations based on path or request protocol.
+
+## API gateway use cases
 
 API gateways solve the following primary use cases:
 
-- **Control access at the point of entry**: Set the protocols of external connection
-  requests and secure inbound connections with TLS certificates from trusted
-  providers, such as Verisign and Let's Encrypt.
-- **Simplify traffic management**: Load balance requests across services and route
-  traffic to the appropriate service by matching one or more criteria, such as
-  hostname, path, header presence or value, and HTTP method.
-
-Consul supports API
-gateways for virtual machines and Kubernetes networks. Refer to the following documentation for next steps:
-- [API Gateways on VMs](/consul/docs/connect/gateways/api-gateway/usage)
-- [API Gateways for Kubernetes](/consul/docs/api-gateway).
+- **Control access at the point of entry**: Set the protocols of external connection requests and secure inbound connections with TLS certificates from trusted providers, such as Verisign and Let's Encrypt.
+- **Simplify traffic management**: Load balance requests across services and route traffic to the appropriate service by matching one or more criteria, such as hostname, path, header presence or value, and HTTP method.
+
+## Workflows
+
+You can deploy API gateways to networks that implement a variety of computing environments: 
+
+- Services hosted on VMs
+- Kubernetes-orchestrated service containers
+- Kubernetes-orchestrated service containers in OpenShift
+
+The following steps describe the general workflow for deploying a Consul API gateways:
+
+1. For Kubernetes-orchestrated services, install Consul on your cluster. For Kubernetes-orchestrated services on OpenShift, you must also enable the `openShift.enabled` parameter. Refer to [Install Consul on Kubernetes](/consul/docs/gateways/api-gateway/install-k8s) for additional information.  
```

Comment:
I think the link should be `consul/docs/connect/gateways/api-gateway/install-k8s`

Reply:
What Adam has is the new URL format where `connect/` is being dropped. The redirect happens in `redirect.js` at the bottom of this PR diff

Reply:
Hmm it doesn't seem to be working in preview mode, should it be? 
Also it looks like we are **adding** connect to the paths in other places in this PR.

Reply:
Probably still good practice to update the link